app/src/main/java/org/thoughtcrime/securesms/ui/theme/Themes.kt

```diff
@@ -0,0 +1,93 @@
+package org.thoughtcrime.securesms.ui.theme
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.selection.LocalTextSelectionColors
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Shapes
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import org.session.libsession.utilities.AppTextSecurePreferences
+
+// Globally accessible composition local objects
+val LocalColors = compositionLocalOf <ThemeColors> { ClassicDark() }
+val LocalType = compositionLocalOf { sessionTypography }
+
+var selectedTheme: ThemeColors? = null
```

Comment:
top-level var is probably a code smell.

Reply:
Yeah I don't love this but wanted something temporary while we get a root composable.
Maybe I can tweak the sharedPrefs flow once they're added in. Though I wouldn't move the compose data outside the app module where it belongs.